sdcm/utils/minicloud.py

```
@@ -0,0 +1,231 @@
+            self.binary_path,
+            "--port",
+            str(self.port),
+        ]
+        if self.lightweight:
+            cmd.append("--lightweight")
+
+        LOGGER.info("Starting minicloud: %s", " ".join(cmd))
+
+        os.makedirs(self.state_dir, exist_ok=True)
+
+        self._process = subprocess.Popen(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)